Packet++/src/GtpLayer.cpp

```diff
@@ -155,522 +155,521 @@ GtpV1Layer::GtpExtension GtpV1Layer::GtpExtension::createGtpExtension(uint8_t* d
 
 GtpV1Layer::GtpV1Layer(GtpV1MessageType messageType, uint32_t teid)
 {
-    init(messageType, teid, false, 0, false, 0);
-
+	init(messageType, teid, false, 0, false, 0);
 }
 
 GtpV1Layer::GtpV1Layer(GtpV1MessageType messageType, uint32_t teid, bool setSeqNum, uint16_t seqNum, bool setNpduNum, uint8_t npduNum)
 {
-    init(messageType, teid, setSeqNum, seqNum, setNpduNum, npduNum);
+	init(messageType, teid, setSeqNum, seqNum, setNpduNum, npduNum);
 }
 
 void GtpV1Layer::init(GtpV1MessageType messageType, uint32_t teid, bool setSeqNum, uint16_t seqNum, bool setNpduNum, uint8_t npduNum)
 {
-    size_t dataLen = sizeof(gtpv1_header);
-    if (setSeqNum || setNpduNum)
-    {
-        dataLen += sizeof(gtpv1_header_extra);
-    }
+	size_t dataLen = sizeof(gtpv1_header);
+	if (setSeqNum || setNpduNum)
+	{
+		dataLen += sizeof(gtpv1_header_extra);
+	}
 
-   	m_DataLen = dataLen;
-	m_Data = new uint8_t[m_DataLen];
-	memset(m_Data, 0, m_DataLen);
+	m_DataLen = dataLen;
+	m_Data = new uint8_t[dataLen];
+	memset(m_Data, 0, dataLen);
 	m_Protocol = GTPv1;
 
-	gtpv1_header* hdr = getHeader();
+	gtpv1_header *hdr = getHeader();
 	hdr->version = 1;
 	hdr->protocolType = 1;
-    hdr->messageType = (uint8_t)messageType;
-    hdr->teid = htobe32(teid);
+	hdr->messageType = (uint8_t)messageType;
+	hdr->teid = htobe32(teid);
 
-    if (setSeqNum || setNpduNum)
-    {
-        hdr->messageLength = htobe16(sizeof(gtpv1_header_extra));
-        gtpv1_header_extra* extraHdr = getHeaderExtra();
-        if (setSeqNum)
-        {
-            hdr->sequenceNumberFlag = 1;
-            extraHdr->sequenceNumber = htobe16(seqNum);
-        }
+	if (setSeqNum || setNpduNum)
+	{
+		hdr->messageLength = htobe16(sizeof(gtpv1_header_extra));
+		gtpv1_header_extra *extraHdr = getHeaderExtra();
+		if (setSeqNum)
+		{
+			hdr->sequenceNumberFlag = 1;
+			extraHdr->sequenceNumber = htobe16(seqNum);
+		}
 
-        if (setNpduNum)
-        {
-            hdr->npduNumberFlag = 1;
-            extraHdr->npduNumber = npduNum;
-        }
-    }
+		if (setNpduNum)
+		{
+			hdr->npduNumberFlag = 1;
+			extraHdr->npduNumber = npduNum;
+		}
+	}
 }
 
 
-bool GtpV1Layer::isGTPv1(const uint8_t* data, size_t dataSize)
+bool GtpV1Layer::isGTPv1(const uint8_t *data, size_t dataSize)
 {
-    if (data != NULL && dataSize > 1 && (data[0] & 0xE0) == 0x20)
-    {
-        return true;
-    }
+	if(data != NULL && dataSize > 1 && (data[0] & 0xE0) == 0x20)
+	{
+		return true;
+	}
 
-    return false;
+	return false;
 }
 
-GtpV1Layer::gtpv1_header_extra* GtpV1Layer::getHeaderExtra()
+GtpV1Layer::gtpv1_header_extra *GtpV1Layer::getHeaderExtra() const
```

Comment:
please move the `*` back to its original place. This is the convention in the rest of the project

Reply:
Done